frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1558,6 +1625,13 @@ impl<T: Config> Pallet<T> {
 				if Self::round() != 1 {
 					log!(info, "Finalized election round with compute {:?}.", compute);
 				}
+				match Self::try_update_minimum_untrusted_score(score) {
```

Comment:
Question: Should we use only successful elections with signed solutions (since we believe that delivers the highest quality of election results) or we take into account every election?